projects/power_distribution/test/test_front_uv_detector.c

```diff
@@ -0,0 +1,55 @@
+#include "can_transmit.h"
+#include "front_uv_detector.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "ms_test_helpers.h"
+#include "pd_events.h"
+
+static GpioAddress s_uv_comp_pin_address = { .port = GPIO_PORT_B, .pin = 0 };
```

Comment:
Use `FRONT_UV_COMPARATOR_PIN` from pin_defs.h instead of hardcoding the value here.